ova/Libraries/wazuh_functions.sh

```diff
@@ -0,0 +1,71 @@
+install_wazuh() {
+
+    set_wazuh_repository
+    install_nodejs
+    yum install wazuh-manager-${WAZUH_VERSION} wazuh-api-${WAZUH_VERSION} -y
+    configure_manager
+    configure_api
+    delete_logs
+}
+
+set_wazuh_repository(){
+    if [ "${STATUS_PACKAGES}" == "stable" ]; then
+    # Wazuh production repository
+        echo -e '[wazuh_repo]\ngpgcheck=1\ngpgkey=https://packages.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=Wazuh repository \nbaseurl=https://packages.wazuh.com/3.x/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh.repo
+
+    fi
+
+    if [ "${STATUS_PACKAGES}" == "unstable" ]; then
+    # Wazuh pre-release repository
```

Comment:
Check the indentation of this line.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/06b971366820c9d9cbf359ba4c3b01c427772310